### PR TITLE
StakeRequest model (with a direct usage of sqlite)

### PR DESCRIPTION
**Pull request by pgev**
Implements DatabaseWrapper and StakeRequestRepository classes.
DatabaseWrapper abstracts away and promisifies database operations.
StakeRequestRepository is a model to work with stake_request database table.

Fixes: #53

### CODE REVIEW

**Comment by deepesh-kn** [comment]
I am in favor of using ORM. `sequelize` looks a good option 👍

**Comment by pgev** [comment]
> I am in favor of using ORM. `sequelize` looks a good option 👍

@deepesh-kn could you please elaborate in which way. ORM as an interface or as an implementation? @sarvesh-ost was thinking (Sarvesh, please correct me here), to use sequelize for implementation to ease manipulations with database. However, ORM is much broader concept, and it's also possible to use it as an interfaces in the project.

File: src/data_access_objects/StakeRequestRepository.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      + `${StakeRequestRepository.nonceColumnName}, `
+      + `${StakeRequestRepository.gatewayColumnName}, `
+      + `${StakeRequestRepository.stakerProxyColumnName} `
+      + ') '
+      + 'VALUES '
+      + '( '
+      + `'${stakeRequest.stakeRequestHash}', `
+      + `'${stakeRequest.messageHash}', `
+      + `${stakeRequest.amount}, `
+      + `'${stakeRequest.beneficiary}', `
```
Comment (benjaminbollen) [inline src/data_access_objects/StakeRequestRepository.ts]: extra single quotes?

Comment (benjaminbollen) [reply src/data_access_objects/StakeRequestRepository.ts]: nope, different types :D

**Pull request merged**